packages/amplify-cli/src/plugin-helpers/platform-health-check.ts

```diff
@@ -0,0 +1,190 @@
+import path from 'path';
+import chalk from 'chalk';
+import { PluginInfo } from '../domain/plugin-info';
+import { PluginPlatform } from '../domain/plugin-platform';
+import { readJsonFileSync } from '../utils/readJsonFile';
+
+type PluginDescription = {
+  name: string;
+  type: string;
+  packageName: string;
+  packageVersion?: string;
+};
+
+const officialPlugins: { [key: string]: PluginDescription } = {
+  core: {
```

Comment:
We could simplify it to be an array or ```PluginDescription```, no need for the nested objects as we don't use the key values for anything, just to iteration.

Reply:
check

Reply:
I kept the key values in the update, because it's much easier to  pinpoint plugins like `officialPlugins.core` (this is used in the codebase) and `officialPlugins.awscloudformation` etc. For future updates, we are going to log the plugin scan result, it will be much eaiser to iterate through the keys which are the just plugin names.